test/test_dlpack.py

```diff
@@ -144,6 +144,17 @@ def __dlpack__(self, stream=None):
             x = DLPackTensor(make_tensor((5,), dtype=torch.float32, device=device))
             from_dlpack(x)
 
+    @skipMeta
+    @onlyCUDA
+    @skipCUDAIfRocm
+    def test_dlpack_convert_default_stream(self, device):
+        x = torch.zeros(2**30, device=device)
+        torch.cuda.default_stream().synchronize()
+        self.assertTrue(torch.cuda.default_stream().query())
+        d = x.__dlpack__(1)
+        # check that the default stream has work
```

Comment:
can you leave a better description what's happening here? Why is there work on default stream, isn't `__dlpack__` supposed to be a zero-copy operation?

Reply:
My understanding is that even if it's zero-copy, the test runs on a different stream by default, so the `zeros` call doesn't complete instantly, and `__dlpack__(1)` has the default stream wait on the current stream.